plugins/weida-raw/weida-raw.quirk

```diff
@@ -0,0 +1,4 @@
+# Raw
+[HIDRAW\VEN_2575]
+Plugin = weida_raw
+GType = FuWeidaRawDevice
```

Comment:
I don't think you need to set the GType if there's only one to choose from.

Reply:
solved